packages/fxa-settings/src/components/CardHeader/index.tsx

```diff
@@ -77,6 +133,49 @@ const CardHeader = (props: CardHeaderProps) => {
     );
   }
 
+  if (isSeparateSubheading(props)) {
+    const { serviceName = MozServices.Default } = props;
+    const isDefaultService = isDefaultServiceName(serviceName);
+    const logo = serviceLogos[serviceName];
+    const logoElem = <span>{logo}</span>;
+
+    const subheadingFtlMsgProps = {
+      id: isDefaultService
+        ? props.subheadingWithDefaultServiceFtlId
+        : props.subheadingWithCustomServiceFtlId,
+      // include `vars={{ serviceName }}` if non-default
+      ...(!isDefaultService && { vars: { serviceName } }),
+      // include `elems={{ span: logoElem }}` if serviceName is given a logo in serviceLogos
+      ...(logo && {
+        elems: { span: logoElem },
+      }),
+    };
+
+    return (
+      <>
+        <h1 className="card-header">
+          <FtlMsg id={props.headingTextFtlId}>{headingText}</FtlMsg>
+        </h1>
+        <FtlMsg {...subheadingFtlMsgProps}>
+          <p className="card-subheader">
```

Comment:
Part of me wonders if this should be an `h2` (same styles of course) or if there's something else a11y-wise we can do here to better associate this with the `h1`.

Reply:
Agreed. I feel like now that all of our different kinds of "card headers" are being documented in Storybook we'll be able to point Product or Content to them and maybe we'll change some of these down the line. I think it's okay as-is right now, just not ideal.